lumper/src/vmm/src/lib.rs

```diff
@@ -160,6 +167,26 @@ impl VMM {
         Ok(())
     }
 
+    pub fn configure_console(
+        &mut self,
+        console_path: Option<String>
+    ) -> Result<()> {
+        if let Some(console_path) = console_path {
+            // We create the file if it does not exist, else we open
+            let file = OpenOptions::new().
+                append(true).
+                create(true).
+                open(console_path).
+                map_err(ConsoleError)?;
+
+            self.serial = Arc::new(Mutex::new(
+                LumperSerial::new(Box::new(file)).map_err(Error::SerialCreation)?,
+            ));
```

Comment:
About these three lines, I feel like it's not something great. I'd rather update the inner value of the Mutex than this.

I found an alternative to this, but I think it's even worse than the current solution:

```rust
            let mut serial = self.serial.lock().unwrap();
            // this works
            mem::replace(&mut *serial, LumperSerial::new(Box::new(file)).map_err(Error::SerialCreation)?);
```


To finish, I tried a third solution, where I was just updating the value by assignation, but I couldn't make it work.

```rust
            let mut serial = self.serial.lock().unwrap();
            // this does not work
            *serial = LumperSerial::new(Box::new(file)).map_err(Error::SerialCreation)?;
```

Reply:
> ```rust
>             let mut serial = self.serial.lock().unwrap();
>             // this does not work
>             *serial = LumperSerial::new(Box::new(file)).map_err(Error::SerialCreation)?;
> ```

This is the idiomatic way of modifying the inner value of a mutex, and it does work. What's not working for you?

Reply:
I was pretty sure it did not work yesterday when I tried. Whatever, now it's working as intended, I did the update.